storage/rocksdb/rdb_compact_filter.h

```diff
@@ -76,6 +76,7 @@ class Rdb_compact_filter : public rocksdb::CompactionFilter {
     } else if (m_ttl_duration > 0 &&
                should_filter_ttl_rec(key, existing_value)) {
       m_num_expired++;
+      global_stats->rows[ROWS_EXPIRED].inc();
```

Comment:
It might be more efficient to upload these stats during deconstruction of the Rdb_compact_filter object to the global stats. Maybe add an api to the ha_rocksdb_proto.h that lets you update a specific counter?